<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogsWindow</class>
 <widget class="QMainWindow" name="LogsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JAY Security - Log Aktivitas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
QWidget {
    background-color: #2c3e50;
    color: #ecf0f1;
    font-family: 'Arial', sans-serif;
}

QLabel {
    font-size: 28px;
    font-weight: bold;
    color: #ecf0f1;
    margin-bottom: 20px;
}

QPushButton {
    background-color: #3498db;
    color: white;
    padding: 10px 20px;
    border-radius: 5px;
    font-size: 16px;
    font-weight: bold;
    border: none;
    transition: background-color 0.3s;
}

QPushButton:hover {
    background-color: #2980b9;
}

QTableWidget {
    background-color: #34495e;
    color: #ecf0f1;
    border: none;
}

QTableWidget::item {
    padding: 10px;
}

QHeaderView::section {
    background-color: #2980b9;
    color: white;
    padding: 10px;
    border: none;
}

QHeaderView {
    border: none;
}

QScrollBar:vertical {
    border: none;
    background: #34495e;
    width: 10px;
    margin: 22px 0 22px 0;
}

QScrollBar::handle:vertical {
    background: #2980b9;
    min-height: 20px;
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    background: none;
    height: 0;
}

QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical {
    background: none;
}
    </string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>

    <!-- Judul untuk Log -->
    <item>
     <widget class="QLabel" name="LogsTitle">
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="text">
       <string>JAY Security - Log Aktivitas</string>
      </property>
     </widget>
    </item>

    <!-- Tabel untuk Menampilkan Log -->
    <item>
     <widget class="QTableWidget" name="logsTable">
      <property name="columnCount">
       <number>5</number>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="horizontalHeaderLabels">
       <stringlist>
        <string>Waktu</string>
        <string>Jenis Serangan</string>
        <string>IP Sumber</string>
        <string>IP Tujuan</string>
        <string>Status Mitigasi</string>
       </stringlist>
      </property>
     </widget>
    </item>

    <!-- Tombol untuk Refresh dan Clear Log -->
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
        <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Refresh Log</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear Log</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>